pom.xml

```diff
@@ -26,7 +26,6 @@
         DO NOT remove these tags but instead fill them in appropriately as needed,
         UNLESS the inherited values are valid for your project.
     -->
-
```

Comment:
just a line is deleted?

Reply:
Hi @Deven-Rawat yes, it was a blank line, so that I could force commit master update, nothing was deleted.